src/style.css

```diff
@@ -1 +1,27 @@
-/* styles go here */
+body {
+  margin: -12px 10px;
+}
+
+fieldset {
+  margin: 20px 0;
+}
+
+.personalInformation > .input {
+  margin: -10px 0;
+  padding: 10px 0;
+}
+
+.registration > .input__r {
+  margin: -10px 0;
+  padding: 10px 0;
+}
+
+.interesting > .input__i {
+  margin: -10px 0;
+  padding: 10px 0;
+}
+
+.additionalInfo > .input__a {
+  margin: -10px 0;
+  padding: 10px 0;
+}
```

Comment:
why do you need all these classes with the same styles?

Reply:
fixed

Reply:
fixed